.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+jobs:
+  test:
+    timeout-minutes: 60
```

Comment:
It actually looks like that step only takes 70 seconds or so and doing caching of the browser binaries (e.g. something along the lines of https://playwrightsolutions.com/playwright-github-action-to-cache-the-browser-binaries/) would not bring any substantial speed ups.

Reply:
That file was autogenerated and I do not care for now for running it as part of the CI/CD pipeline. My top priority is to fix the Python code, so that I am not blocking @zdaratom when he returns.

I am skipping this for now.